packages/styles/src/components/alert.scss

```diff
@@ -5,206 +5,128 @@
-  }
-}
+  @include media-breakpoint-down(md) {
+    flex-direction: column;
+    align-items: stretch;
+    padding-inline: map.get(notification.$notification-padding-x-map, alert);
 
-.alert-heading {
-  @extend %notification-title;
+    > .alert-content {
+      $icon-size: map.get(notification.$notification-icon-size-map, alert);
 
-  font-size: alert.$alert-font-size;
-}
+      padding-inline-start: notification.$notification-gap + $icon-size;
+      padding-inline-end: notification.$notification-gap + close.$close-size;
+    }
 
-@each $name, $color, $icon in alert.$alert-list {
-  .alert-#{$name} {
-    --post-bg-rgb: #{color-fn.rgb-values($color)};
-    @extend %notification-#{$name};
+    > .alert-buttons {
+      display: flex;
 
-    @if (contrast-fn.light-or-dark($color) == 'dark') {
-      @extend %color-background-dark-variables;
-    } @else {
-      @extend %color-background-light-variables;
-    }
+      > .btn {
+        flex: 1 0 0;
+        width: 100%;
 
-    .btn-close {
-      @extend %notification-#{$name}-close;
+        &:first-child {
+          margin-inline-start: 0;
+        }
+      }
     }
   }
 }
 
 .alert-fixed-bottom {
   position: fixed;
-  z-index: commons.$zindex-alert;
-  bottom: 0;
-  left: 0;
-  width: 100%;
-  max-height: 100%;
-  overflow: auto;
+  inset-block-end: 0;
+  inset-inline-start: 0;
+  inset-inline-end: 0;
   margin-bottom: 0;
   border-radius: 0;
-
-  // Let the icon scroll with the content
-  background-attachment: local;
-
-  @include media-breakpoint-up(xxl) {
-    $container-padding: map.get(grid.$grid-container-padding, 'xxl');
-    padding: alert.$alert-padding-y
-      calc(
-        (100vw - #{map.get(breakpoints.$grid-breakpoints, 'xxl')}) / 2 + #{alert.$alert-padding-x *
-          2 + alert.$alert-icon-size} + #{$container-padding} - 0.5rem
-      );
-
-    background-position: calc(
-        (100vw - #{map.get(breakpoints.$grid-breakpoints, 'xxl')}) / 2 + #{alert.$alert-padding-x} +
-          #{$container-padding} - 0.5rem
-      )
-      center;
-
-    &.alert-dismissible {
-      padding-right: calc(
-        (100vw - #{map.get(breakpoints.$grid-breakpoints, 'xxl')}) / 2 + #{alert.$alert-close-icon-size +
-          alert.$alert-padding-x * 2} + #{$container-padding} - 0.5rem
-      );
-
-      .btn-close {
-        right: calc(
-          (100vw - #{map.get(breakpoints.$grid-breakpoints, 'xxl')}) / 2 + #{$container-padding} - 0.5rem
-        );
-      }
-    }
-  }
-}
-
-.alert-action {
-  flex-direction: row;
-  flex-wrap: nowrap;
-  align-items: center;
-  justify-content: space-between;
-  gap: spacing.$size-regular;
-
-  @include media-breakpoint-down(lg) {
-    flex-direction: column;
-    align-items: flex-start;
-    justify-content: flex-end;
-
-    .alert-buttons {
-      align-self: flex-end;
-    }
-  }
-
-  @include media-breakpoint-down(sm) {
-    align-items: center;
-  }
+  z-index: commons.$zindex-fixed;
 
   @include media-breakpoint-up(xxl) {
```

Comment:
@oliverschuerch 
With my last two commits, I removed CSS variables and used mixins instead.
It works well for the majority of styles, except here where I had to replace many properties with calculations.
What do you think?